agents/processor/src/push.rs

```diff
@@ -42,11 +40,7 @@ impl std::fmt::Debug for Pusher {
 impl Pusher {
     /// Instantiate a new pusher with a region
     pub fn new(name: &str, bucket: &str, region: Region, db: NomadDB) -> Self {
-        let client = S3Client::new_with(
-            HttpClient::new().unwrap(),
-            EnvironmentProvider::with_prefix(AWS_S3_PREFIX),
-            region.clone(),
-        );
+        let client = S3Client::new(region.clone());
```

Comment:
@arnaud036 note that with this change S3 and KMS now always use the same credentials. No more need for the `OPT_PROCESSOR_S3_*` env vars